environment/src/main/resources/schema/app/20220303161846_CB-15814_Option_to_create_without_load_balancer.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-15814 Add option
+-- Migration SQL that makes the change goes here.
+
+
+ALTER TABLE environment_network ADD IF NOT EXISTS loadBalancerCreation varchar(255) DEFAULT 'ENABLED' NOT NULL;
```

Comment:
The name `loadBalancerCreation` will be converted to a lower case one, I think it would make sense to have it on the same way in the SQL as it will be added as column name `loadbalancercreation`.